misk-aws-dynamodb-testing/build.gradle

```diff
@@ -0,0 +1,21 @@
+dependencies {
+  compile dep.awsApi
```

Comment:
`api dep.awsApi`?

Reply:
I don't know what that does... This was a mostly copy paste from the `misk-aws` `build.gradle`

Reply:
> The plugin exposes two configurations that can be used to declare dependencies: api and implementation. The api configuration should be used to declare dependencies which are exported by the library API, whereas the implementation configuration should be used to declare dependencies which are internal to the component.

https://docs.gradle.org/current/userguide/java_library_plugin.html#sec:java_library_separation

Reply:
because this library exposes `AmazonDynamoDB`, it should have API dependency on dep.awsApi. Since this is copy paste let's leave it as is and change it later?

Reply:
👍